chanzuckerberg/single-cell-data-portal
# feat: Implement nav bar button with link to read the docs site for Cell Census (#4615)

## Reason for Change
A landing page for the Census should exist at cellxgene.cziscience.com.

- #TICKET_NUMBER
- #4615
- #4624

## Changes

- Added "Census" to site navigation bar as per [mock](https://www.figma.com/file/DMoBg0zsvktKrOZ3akAnFc/Cell-Census?node-id=144-20144&t=7TRCwr77V9EDW4oU-0). Note, this mock is now deprecated. The requirement for the link text changed from "Cell Census" to "Census". See [slack](https://clevercanary.slack.com/archives/C02CF7MQPGV/p1681838047601139?thread_ts=1681775062.031959&cid=C02CF7MQPGV) for discussion.
  - "Census" should open in the current tab. See [slack](https://clevercanary.slack.com/archives/C02CF7MQPGV/p1681837807824659?thread_ts=1681775062.031959&cid=C02CF7MQPGV) for discussion.
  - Created a new navigation bar component, providing one source of truth for the left hand side navigation links. (Note: the new navigation bar component is without responsive behaviour, however, the mobile sidebar navigation remains intact for the landing page, as per production.). See slack for discussion [here](https://clevercanary.slack.com/archives/C02CF7MQPGV/p1681858774558289?thread_ts=1681795727.766849&cid=C02CF7MQPGV).
  - Duplicated navigation bar related styles have been removed.
  - Removed unused, duplicated styles from landing page header.
  - Updated duplicated styles as per slack discussion [here](https://clevercanary.slack.com/archives/C02CF7MQPGV/p1681847673496079?thread_ts=1681795727.766849&cid=C02CF7MQPGV) and [here](https://clevercanary.slack.com/archives/C02CF7MQPGV/p1681858774558289?thread_ts=1681795727.766849&cid=C02CF7MQPGV) and [here](https://clevercanary.slack.com/archives/C02CF7MQPGV/p1681876403926769?thread_ts=1681795727.766849&cid=C02CF7MQPGV). Note, these style updates are not yet de-duped. They are out of scope of this ticket.
  - Note: the navigation styles are unchanged (left side and right side) e.g. fonts/weights/colours - this is out of scope of this ticket. See slack discussion [here](https://clevercanary.slack.com/archives/C02CF7MQPGV/p1681931792194909?thread_ts=1681795727.766849&cid=C02CF7MQPGV).
  - The "We're Hiring!" button is removed as per slack thread [here](https://clevercanary.slack.com/archives/C02CF7MQPGV/p1681860775200339).
- Added analytics to the new "Census" link as per #4624.

## Testing steps

- Go to the landing page.
  - "Census" link displays after the divider and "Gene Expression".
  - Landing page should not display "We're Hiring!" button.
  - Confirm mobile sidebar navigation the link "Census" displays below the divider.
- Click on the "Census" link. The "Census" page should load in the current tab.
- Go to the Collections page.
  - "Census" link should display as per landing page.
  - Confirm header and navigation renders without responsive behaviour .
